commit b47f952a8b6823e40da84dac916abe34e416e705 (refs/changes/70/4570/2)
Author: Rui Loura <rui@joyent.com>
Date:   2018-07-25T20:06:22+00:00 (1 year, 2 months ago)
    
    MANTA-3753 muppet should leverage autometadata instead of checking nictags (bump version)
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Approved by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/package.json b/package.json
index da15204..3c0f21b 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
     "name": "muppet",
     "description": "Joyent's Load Balancer",
-    "version": "1.0.1",
+    "version": "1.1.0",
     "author": "Joyent (joyent.com)",
     "private": true,
     "dependencies": {
